docs/pages/blog/aggregation-functions.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Aggregate data like in Excel, but easier!
+description: Introducing aggregation functions and summary rows in Data grid Premium.
+date: 2022-08-01T00:00:00.000Z
+authors: ['josefreitas', 'flaviendelangle', 'cherniavskii']
+tags: ['MUI X', 'News']
+card: false
+---
+
+If you ever had a lot of data on a grid, you probably know that the ability to set different perspectives on data is essential to help users extract information. Typically, one would use at least filtering or sorting, but that barely scratches the surface of what one could do.
+
+The Data grid Premium is all about enabling and empowering advanced use cases for data analysis.
+And in that spirit, we’re overjoyed to announce that starting from [v5.15.0](https://github.com/mui/mui-x/releases/tag/v5.15.0), you can now use [aggregation functions](/x/react-data-grid/aggregation/), and with a couple of clicks, extract information like **sum**, **average**, **count**, and others.
+
+## Wait, what is an aggregation function?
+
+In a nutshell, those are functions used to return a computed result based on the values of a given column. For example, you have a list of financial transactions, and you want to get the Sum of all transactions and display it in a summary row.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/aggregation-functions/summary-row.mp4" type="video/mp4" />
+</video>
+
+## High end-user experience
+
+The **Sum** function showing a `Total` in summary rows is probably one of the most common use cases. But users can employ multiple aggregation functions in different columns simultaneously, and even combine with row grouping to aggregate children’s data on each group.
```

Comment:
Couple tiny style changes here. Also, is there another way to say "aggregate children's data on each group"? It's not entirely clear to me what this means.

```suggestion
The **Sum** function showing a **Total** in summary rows is one of the most common use cases. But users can employ multiple aggregation functions in different columns simultaneously, and even combine with row grouping to aggregate children’s data on each group.
```